taskfile.yml

```diff
@@ -151,3 +152,13 @@ tasks:
       vars: [cmd]
     cmds:
       - "{{ .SAIL }} {{ .cmd }}"
+
+  get-sail:
+    desc: Get Laravel Sail
+    cmds:
+      - test -f 'vendor/bin/sail' && echo "Sail script exists, skip" || docker run --rm
+        -u "$(id -u):$(id -g)"
+        -v "$(pwd):/var/www/html"
+        -w /var/www/html
+        laravelsail/php84-composer:latest
```

Comment:
In the current composer.json, we require `php: ^8.2`, which might fail if we install it with version 8.4. Is there another image available?

Reply:
Good spot, thank you.